validation/validation.go

```diff
@@ -74,3 +78,10 @@ func validateIssueTrackerExists(cfg *config.Local, tracker issuetracker.IssueTra
 
 	return fmt.Errorf("repository %s not found", cfg.Origin)
 }
+
+func validateTrackerAuthType(cfg *config.Local) error {
```

Comment:
Could you add unit tests for this new validation in `validation/validation_test.go`?